telemetry/definitions/commonDefinitions.json

```diff
@@ -2841,6 +2841,10 @@
                 {
                     "type": "codeTransformSessionId",
                     "required": true
+                },
```

Comment:
Here is the definition for that type:

```
        {
            "name": "enabledAuthConnections",
            "type": "string",
            "description": "Comma delimited list of enabled auth connections"
        },
        ```

Reply:
Discussed in Slack: going to use `jobIsStartedFromChatPrompt` for now since `projectDetails` is always emitted on IDE startup; we want this metric to capture just users who tried transforming a valid project.